include/aspect/initial_conditions/S40RTS_perturbation.h

```diff
@@ -31,7 +31,7 @@ namespace aspect
   {
     using namespace dealii;
 
-    namespace internal
+    namespace S40RTSinternal
```

Comment:
I found that S40RTS and SAVANI cannot have the same namespace "internal", that's why in the Hackathon the SAVANI test cannot finished with 600s. It seems they got some "mess-up" problem if they use the same namespace. S40RTSinternal and SAVANIinternal are similar, but not the same, because their data structures are different. In addition to the different spherical harmonic coefficients, the center interpolation depth and the number of layers are also different (SAVANI-28, S40RTS-21). I once thought I can add if loop in the code to include two initial conditions, but I suppose it's not so elegant- One source file containing two initial condition. Also, in the future, what if we add other tomography model using different interpolation methods? I think InitialConditions::S40RTSinternal/SAVANIinternal seems good. Or can we replace S40RTSinternal directly with S40RTS and the same with SAVANI?